src/components/StatueList.spec.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import StatueList from "./Statuelist";
+
+it("renders without crashing", () => {
+  const MockSlaverStatues = [
+    {
+      statueName: "Edward Colston",
+    },
+  ];
```

Comment:
Worth either using a proper factory package for this, or at least using this into a factory file.

Reply:
We will do this for the next test